src/Resources/config/services.xml

```diff
@@ -15,7 +15,7 @@
 
         <service id="cmf_seo.form.type.seo_metadata" class="Symfony\Cmf\Bundle\SeoBundle\Form\Type\SeoMetadataType">
             <argument>%cmf_seo.form.data_class.seo_metadata%</argument>
-            <argument>%cmf_seo.form_mode_orm%</argument>
+            <argument>%cmf_seo.form.options%</argument>
             <tag name="form.type" alias="seo_metadata" />
```

Comment:
if we would not, we could not use a constructor afaik. see http://symfony.com/doc/current/form/create_custom_field_type.html#creating-your-field-type-as-a-service